docs/getting-started/accessing-resources/getting-access-to-data.md

```diff
@@ -247,46 +265,69 @@ To list all available releases, you can use the following command from the root
+Running either of the above commands will update the `data/current` symlink to point to the `data/test` directory.
 This means any real ScPCA data or results you had previously downloaded will no longer be in the `data/current` path.
-To switch this symlink path back to ScPCA data or results, rerun either script with the `--release current` option:
+You can use the `--update-symlink` flag, as described below, to update which release directory the `data/current` symlink directs to.
 
-```sh
-./download-data.py --release current
+## Updating the `current` symlink
+
+As described above, the `data/current` directory is a [symlink](https://en.wikipedia.org/wiki/Symbolic_link) to the most recently-downloaded release directory.
+You can update this symlink to point to a different data release directory (or to the test data directory) in one of two ways:
+
+1. When you download a new data or results release, `data/current` will automatically be updated to direct to the newly-downloaded release directory.
+    - Both the `download-data.py` and `download-results.py` scripts will print a message telling you where `current` directs to.
+1. You can use the `download-data.py` script with the `--update-symlink` flag to redirect the `data/current` symlink to a release of your choice.
+Note that no data will be downloaded if you use this flag.
+Below are some common use cases for this flag:
+
+    ```sh
+    # By default, this flag will update the current symlink to direct to the
+    #  most recent release that is present on your computer
+    ./download-data.py --update-symlink
+
+    # Use the --release option to specify which release to direct the symlink to
+    ./download-data.py --update-symlink --release 2025-05-01
+
+    # Use the --test-data flag to direct the symlink to the test data
+    ./download-data.py --update-symlink --test-data
+    ```
+
+
+## Accessing data from the ScPCA Portal
 
-./download-results.py --release current
+If you have not formally joined the OpenScPCA project, you will need to obtain data directly from ScPCA Portal that the Data Lab maintains: <https://scpca.alexslemonade.org/>.
+
+You can select the project(s) or sample(s) you are interested in analyzing and download them from the Portal.
+
+We recommend creating a `portal-downloads` subdirectory in the local copy of the `data` directory, which can be accomplished by running the following command from the root of the repository on Linux or macOS:
+
+```sh
+mkdir -p data/portal-downloads
 ```
-If you had already downloaded the most recent data or results, this will not repeat downloading the files you already have.
 
--->
\ No newline at end of file
+You can then develop your analysis using these paths.
+
+However, before filing a pull request, you should change your paths to use `data/current` directory [established by the data download script](#download-data-file-structure).
```

Comment:
If they set up a symlink, they can set their paths up properly the first time.

Which does suggest that we should be explicit about how they should organize files from portal downloads within this directory.

Finally, we should mention that the test data can be downloaded without an account as well.

Reply:
> Finally, we should mention that the test data can be downloaded without an account as well.

We do have this here, but it sounds like this was not so findable! I'll rework a bit so this is clearer. https://github.com/sjspielman/OpenScPCA-analysis/blob/d0ee8d17b3f2177930d778db0aeb7b804bbb95f3/docs/getting-started/accessing-resources/getting-access-to-data.md?plain=1#L276

Reply:
Yes, I know it is in there, but I think it makes sense to add a mention here as well as a reminder.